site/_js/main.js

```diff
@@ -19,13 +19,16 @@ import './store.js';
 
 // Web Components
 // These are components that appear on _every_ page.
+import './web-components/card-section';
 import './web-components/search-box';
 import './web-components/side-nav';
 import './web-components/top-nav';
 import './web-components/share-button';
 import './web-components/web-tabs';
 import './web-components/language-select';
 import './third-party/announcement-banner/announcement-banner';
+import './slider';
```

Comment:
Please don't include `/100/` specific code in the `main` bundle.

Reply:
Done. Moved to /100/ directory.